lib/pharos/configuration/bastion.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Configuration
+    class Bastion < Pharos::Configuration::Struct
```

Comment:
could this actually just be an instance of `Host`?

Reply:
IMO no, at least without heavy refactoring. `Host` has lot of other stuff in it too which we do not need/want at `Bastion`. And having both as `Host` would also make it as never ending recursion :)